api/src/org/apache/cloudstack/api/command/user/vpn/AddVpnUserCmd.java

```diff
@@ -124,7 +124,7 @@ public void execute() {
                 throw new ServerApiException(ApiErrorCode.INTERNAL_ERROR, "Failed to add vpn user");
             }
         }catch (Exception ex) {
```

Comment:
@jayapalu do we need this catch-all here?
It seems that we can simply catch `ResourceUnavailableException`, and then re-throw the exception.
Then, we do not need to do anything with the exception thrown at line 124 does, just let it go.